api/api.go

```diff
@@ -90,6 +90,10 @@ func NewAPIWithVersion(ctx context.Context, globalConfig *conf.GlobalConfigurati
 		r.Get("/", api.ExternalProviderCallback)
 		r.Post("/", api.ExternalProviderCallback)
 	})
+	// Add route for step 2: Takes in Auth Code + Verifier -> Returns token
+	r.Route("/oauth", func(r *router) {
+		r.Post("/token", api.TokenVerifier)
```

Comment:
Shouldn't this be the existing `/token` endpoint?

Reply:
You're right - this is probably a better fit as an [`auth_code` grant_type](https://oauth.net/2/grant-types/) under the existing endpoint